piet-wgsl/examples/winit/Cargo.toml

```diff
@@ -14,3 +14,138 @@ winit = "0.27.5"
 pollster = "0.2.5"
 # for picosvg
 roxmltree = "0.13"
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+console_error_panic_hook = "0.1.7"
+console_log = "0.2"
+wasm-bindgen-futures = "0.4.33"
+js-sys = "0.3.60"
+web-sys = { version = "0.3.60", features = [
+    "Document",
```

Comment:
Why do we need to specify all the GPU specific things? Does `wgpu` not enable those features for us?

Reply:
Removed these. Works fine without them.